Tasks/Xcode/xcode.ts

```diff
@@ -85,6 +85,29 @@ async function run() {
             }
         }
 
+        let destinations: string[];
+        let platform: string = tl.getInput('platform', false);
+        if (platform === 'macOS') {
+            destinations = ['platform=macOS'];
+        }
+        else if (platform && platform !== 'default') {
+            // To be yaml friendly, destinationType is optional and we default to simulators.
+            let destinationType: string = tl.getInput('destinationType', false);
+
+            let devices: string[];
+            let targetingSimulators: boolean;
```

Comment:
It might be cleaner and more legible to put the boolean evaluation on this line:
`let targetingSimulators: boolean = destinationType !== 'devices';`